src/components/questions/question.js

```diff
@@ -1,26 +1,50 @@
 import React from "react";
 import AnswerList from "./answersList"
+import Modal from "../Modal.jsx"
+import AddAnswer from "./addAnswer.js"
 
 
 class Question extends React.Component{
   constructor(props){
     super(props);
-    this.state = {};
+    this.state = {
+      showModal: false
+    };
+    this.helpfulClick = this.helpfulClick.bind(this);
+    this.addAnswerClick = this.addAnswerClick.bind(this);
+  }
+
+  helpfulClick(event){
```

Comment:
Interested in how you will track helpfulness click. Do you need to track individual user to make sure they don't report helpfulness twice.

Reply:
yes sir, i believe mayanne has handled this already so i may talk to her about her implementation